crates/api_common/src/site.rs

```diff
@@ -448,14 +448,10 @@ pub struct GetSiteResponse {
   pub site_view: SiteView,
   pub admins: Vec<PersonView>,
   pub version: String,
-  #[cfg_attr(feature = "full", ts(optional))]
+  #[cfg_attr(feature = "full", ts(skip))]
   pub my_user: Option<MyUserInfo>,
```

Comment:
This should just be removed now right? Or just add deprecated like the other fields, and always have it be None in the code.

Reply:
This is used for get_site_v3 below.